languages/rust/concepts/structs/about.md

```diff
@@ -1 +1,94 @@
-TODO: add information on structs concept
+# Structs
+
+Structs collect a fixed, heterogenous set of data into a single unit. Structs in Rust have these properties:
+
+- Every struct type has a particular name.
+- Every member of a struct is called a **field**.
+- Every field of a struct has a particular name.
+- Visibility is tracked at the field level: each field may be independently public, private, or other[^1].
+
+# Tuples etc.
+
+Structs are very much like **Tuples**. In fact, the only difference between structs and tuples has to do with what is anonymous. The general form goes like this:
+
+- Structs are named, and have named fields:
```

Comment:
I'm honestly fine with these sentences as-is since they're not going to be misunderstood, but from my understanding of rules for using commas, there wouldn't be one here:

* the list has only two elements (whereas you would use commas with "structs are named, have a definite size, and have named fields")
* the "have named fields" doesn't have both a subject and verb (whereas you would use a comma with "structs are named, and they have named fields")

```suggestion
- Structs are named and have named fields:
```

Since I'm no grammar authority, I have no complaints if this suggestion is just ignored or answered with "there is this rule that says you should use a comma here" or "even though it's strictly speaking against the rules to use a comma here, let's use one anyway to make the sentence more readable"